llvm/docs/GitHub.rst

```diff
@@ -23,6 +23,13 @@ intended to be able to support "stacked" pull-request. Do not create any branche
 llvm/llvm-project repository otherwise, please use a fork (see below). User branches that
 aren't associated with a pull-request **will be deleted**.
 
+Note: this is particularly necessary if using [Graphite](https://app.graphite.dev/).
```

Comment:
Introducing Graphite in a "note" seems strange to me. Seems like there should be a section about Graphite itself instead?